addons/events/fnc_addPlayerEventHandler.sqf

```diff
@@ -42,69 +44,86 @@ params [["_type", "", [""]], ["_function", {}, [{}]], ["_applyRetroactively", fa
 
 _type = toLower _type;
 
+// Skip retroactive execution if no current unit.
+if (isNull (missionNamespace getVariable [QGVAR(oldUnit), objNull])) then {
```

Comment:
```suggestion
if (_applyRetroactively && {isNull (missionNamespace getVariable [QGVAR(oldUnit), objNull])}) then {
```
get rid of isNull, missionNamespace, getVariable, objNull
calls in most cases, I'd argue that most users have retro disabled

Reply:
I like it.